english_style_guide.md

```diff
@@ -0,0 +1,97 @@
+_This document has been adapted from
+https://github.com/raywenderlich/english-style-guide_
+
+-------------------------------------------------------------------------------
+
+# English Style Guidelines
+
+This is an English style guide that Hack Club documentation should follow to
+stay consistent in how we refer to common things.
+
+**Table of Contents**
+
+- [English Style Guidelines](#english-style-guidelines)
+  - [Terminology and Capitalization](#terminology-and-capitalization)
+    - [General Guides](#general-guides)
+    - [List of Terminology](#list-of-terminology)
+  - [Punctuation](#punctuation)
+  - [Other Style Guidelines](#other-style-guidelines)
+
+## Terminology and Capitalization
+
+### General Guides
+
+**Company names** Company names are proper nouns and should be capitalized
+according to how the company capitalizes it's name.
+
+### List of Terminology
+
+**club leader** Not _Hack Club leader_ or _club director_.
```

Comment:
So when I see:

> **club leader** Not _Hack Club leader_ or _club director_.

I understand what it's trying to tell me: Do X not Y or Z.

But when "**pop-up**" is listed, I'm not sure what it's trying to say. I can interpret it in the following ways

- "use the word 'pop-up'"
- "spell it 'pop-up' and NOT 'popup' or 'pop up'"

but I'm not sure.

### Content Change Proposal

Instead, I propose in addition to listing _what_ to do, I propose _always_ listing what NOT to do. So for our 'pop-up' example, we would write:

```md
**pop-up** Not popup or pop up
```

### Formating Change Proposal

Moreover than just suggesting a content change, I want to propose a formatting change. I propose we write the "List of Terminology" in the following format:

| Accepted Word | Unaccepted Synonyms                        |
|---------------|--------------------------------------------|
| facilitator   | club leader, lab director, leader, teacher |
| hacker        | learner, student                           |
| OK            | okay, Ok                                   |
| URL           | url, web address                           |
| right-click   | right click, alt click                     |

Thoughts?